vm/src/lib.rs

```diff
@@ -48,6 +49,7 @@ impl Vm {
         pest::state(input, |state| self.parse_rule(rule, state))
     }
 
+    #[allow(clippy::almost_complete_letter_range)]
     fn parse_rule<'a, 'i>(
```

Comment:
@pest-parser/triage One other possibility for this would be to introduce a new `pub fn match_range_inclusive(mut self: Box<Self>, range: RangeInclusive<char>) -> ParseResult<Box<Self>>` and use that in this function, and mark the old `match_range` as deprecated:

https://github.com/pest-parser/pest/blob/df487a6841c5d9e2990d13c4cc1254611efa3912/pest/src/parser_state.rs#L685